### PR TITLE
fluentd monitoring

### PR DESCRIPTION
**Pull request by tamir-michaeli**
* added fluentd-prometheus plugin
* added daemonset & configmap templates for monitoring fluentd metrics.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,13 @@ For container runtime Containerd:
 kubectl apply -f https://raw.githubusercontent.com/logzio/logzio-k8s/master/logzio-daemonset-containerd.yaml -f https://raw.githubusercontent.com/logzio/logzio-k8s/master/configmap.yaml
 ```
 
+For monitoring fluentd:
+add the suffix `-monitoring` to the desired configmap and daemonset templates.
```
Comment (mirii1994) [inline README.md:71]: I think this instruction may be confusing. Maybe it should explicitly say something like "To monitor fluentd on cluster with containerd cri - use the following ............. for monitor fluentd on cluster with docker cri - use the following......."

File: logzio-daemonset-containerd-monitoring.yaml [hunk 2]
```diff
@@ -0,0 +1,133 @@
+---
```
Comment (mirii1994) [inline logzio-daemonset-containerd-monitoring.yaml:1]: Can we add here as a comment that this ds export fluentd metrics?

File: logzio-daemonset-rbac-monitoring.yaml [hunk 3]
```diff
@@ -0,0 +1,133 @@
+---
```
Comment (mirii1994) [inline logzio-daemonset-rbac-monitoring.yaml:1]: Can we add here as a comment that this ds export fluentd metrics?

**Review by mirii1994 — CHANGES_REQUESTED** [review]
Great work, please see my comments :)

**Review by mirii1994 — APPROVED** [review]
Awesome

**Review by Doron-Bargo — APPROVED** [review]
lgtm

**Pull request merged**